src/disk_interface.cc

```diff
@@ -222,6 +222,32 @@ TimeStamp RealDiskInterface::Stat(const string& path, string* err) const {
 }
 
 bool RealDiskInterface::WriteFile(const string& path, const string& contents) {
+#ifdef _WIN32
+  wstring fullpath;
+  string err;
+
+  if (!FullPathName(&fullpath, path, &err)) {
+    Error("WriteFile(%s): %s", path.c_str(), err.c_str());
+    return false;
+  }
+
+  HANDLE handle = CreateFileW(fullpath.c_str(), GENERIC_WRITE, 0, NULL, CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, NULL);
```

Comment:
Can I ask you to use newline so that this line (and some other places) does not exceed 80 chars?